### PR TITLE
Disable Stackdriver debugger when source context is missing

### PR DESCRIPTION
**Pull request by shantuo**

### CODE REVIEW

**Comment by shantuo** [comment]
I do this for now to disable the debugger when user wants to build the image locally. @broady @zombiezen

File: base/third_party/go-wrapper/go-wrapper [hunk 1]
```diff
@@ -86,21 +86,25 @@ case "$cmd" in
...
+			if [ -n "${GAE_SERVICE}" ]; then
+				service="${GAE_SERVICE}"
+			else
+				service="default"
+			fi
+			if [ -n "${GAE_VERSION}" ]; then
+				version="${GAE_VERSION}"
+			else
+				version="default"
+			fi
```
Comment (broady) [inline base/third_party/go-wrapper/go-wrapper]: just:
```
GAE_VERSION=${GAE_VERSION:-default}
GAE_SERVICE=${GAE_SERVICE:-default}
```

**Comment by broady** [comment]
LGTM

File: base/third_party/go-wrapper/go-wrapper (hunk 1 above)
Comment (zombiezen) [inline base/third_party/go-wrapper/go-wrapper]: My only addition would be to quote the rvalue.

**Comment by shantuo** [comment]
PTAL

**Comment by zombiezen** [comment]
LGTM

**Pull request merged**